Sources/WalletConnectPush/Client/Common/DeletePushSubscriptionSubscriber.swift

```diff
@@ -31,7 +29,9 @@ class DeletePushSubscriptionSubscriber {
                 logger.debug("Peer deleted subscription")
                 let topic = payload.topic
                 networkingInteractor.unsubscribe(topic: topic)
-                pushSubscriptionStore.delete(forKey: topic)
+                Task(priority: .high) {
+                    try await pushSubscriptionStore.delete(id: topic)
+                }
```

Comment:
so if client 1A receives delete request from client 2
1A will publish sync delete 
then 1B will receive delete request from client 2 too and then sync delete from 1A?
so 1B will remove the subscription and publish sync delete too?
and then 1A receives sync delete from 1B for already removed subscription?

Reply:
Sync delete only sends if state was changed (SyncStore line 78)